### ISSUE TITLE
The certificate of '3mdeb.com' is not trusted/yet activated

### ISSUE THREAD

**Issue by pietrushnic** [Dasharo/dasharo-issues#894]
### Component

Dasharo Tools Suite

### Device

PC Engines APU2

### Dasharo Tools Suite version

v1.2.21

### Brief summary

errors while testing FDT1.1 (flash using DTS)

### How to reproduce

Use FDT1.1 from old PC Engines testing framework.

### Expected behavior

No errors while getting firmware from 3mdeb.com.

### Actual behavior

```
apu2/apu2_v4.16.0.4.romttps://3mdeb.com/open-source-firmware/pcengines/
--2022-04-28 17:42:47--  https://3mdeb.com/open-source-firmware/pcengines/apu2/apu2_v4.16.0.4.rom
Resolving 3mdeb.com... 178.32.205.96, 2001:41d0:301:5::26
Connecting to 3mdeb.com|178.32.205.96|:443... connected.
ERROR: The certificate of '3mdeb.com' is not trusted.
ERROR: The certificate of '3mdeb.com' is not yet activated.
The certificate has not yet been activated
bash-5.1#
```

**Comment by macpijan** [comment]
Can it be some RTC issue?

`The certificate has not yet been activated` sounds like the time may be incorrect?

**Comment by pietrushnic** [comment]
@miczyg1 are we passing FDT1.1 now? Maybe it is no longer an issue?

**Comment by miczyg1** [comment]
Yes, we are passing. But depending on RTC state we may not be able to boot DTS over iPXE with HTTPS.

For now I succeeded in FDT, btu failures were various: from different iPXE boot menus, to fetching non-existent firmware from dl.3mdeb.com FTP

**Comment by pietrushnic** [comment]
We have to solve those one by one. I was thinking about LAN mirrors, one of the most popular things to avoid networking issues external to us (or at least to limit).
